Unix/http/httpauth.c

```diff
@@ -1529,9 +1538,12 @@ MI_Boolean IsClientAuthorized(_In_ Http_SR_SocketData * handler)
 
                 (* _g_gssState.Gss_Delete_Sec_Context)(&min_stat, &context_hdl, NULL);
 
-                handler->pAuthContext = NULL;
+                handler->pAuthContext  = NULL;
                 handler->authFailed = TRUE;
 
+                (* _g_gssState.Gss_Release_Cred)(&min_stat, handler->pVerifierCred);
```

Comment:
No check if `handler->pVerifierCred` is null or not. How does Gss_Release_cred behave in this case? I also noted that `gss_release_cred` is called without checking the return value. This is done in other cases too, for other gss APIs. Is this a good idea? Even if you just note the error and log it and then continue, having this in the log can provide critical information if things go awry.